operations/host_spawn.go

```diff
@@ -607,6 +627,18 @@ func hostSSH() cli.Command {
 			if key != "" {
 				args = append(args, "-i", key)
 			}
+
+			// Append the arbitrary additional SSH args unless it's the special
+			// case where it's a single arg and it's the host ID.
+			if c.NArg() > 1 || c.NArg() == 1 && c.Args().Get(0) != hostID {
+				args = append(args, c.Args()...)
+			}
+
+			if dryRun {
+				fmt.Printf("Going to execute the following command: %s\n", strings.Join(args, " "))
```

Comment:
can the command be on a new line?

Reply:
Done.

I also did this for the rsync command to keep it consistent.